dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/JacksonEmptyContainerLoader.java

```diff
@@ -150,7 +154,9 @@ private static int decrement(int maxRecursion, TypeMarker<?> originalType) {
         try {
             return Optional.ofNullable(method.invoke(null, parameter));
         } catch (IllegalAccessException | InvocationTargetException e) {
-            log.debug("Reflection instantiation failed", e);
+            if (log.isDebugEnabled()) {
+                log.debug("Reflection instantiation failed", e);
+            }
```

Comment:
No allocation in this one, but it's future-proof :+1:

Reply:
yeah, I may have over-optimized in applying some human automation